ra/ra.go

```diff
@@ -1274,6 +1292,12 @@ func (ra *RegistrationAuthorityImpl) UpdateRegistrationContact(ctx context.Conte
 		return nil, fmt.Errorf("invalid contact: %w", err)
 	}
 
+	// TODO(#7966): Remove once the rate of registrations with contacts has
+	// been determined.
+	for range req.Contacts {
+		ra.newOrUpdatedContactCounter.With(prometheus.Labels{"new": "false"}).Inc()
+	}
+
```

Comment:
Since we're re-reviewing anyway, can this move to be below the call to ra.SA.UpdateRegistrationContact, to match the placement of this stanza in NewRegistration?

Reply:
Certainly!